File: Server/Factory/LoginInfo.js

class LoginInfo {

    constructor(stateCode, msg) {

        // 0 正确 1 密码错 2 用户名错
        this.stateCode = stateCode;

        this.msg = msg;

        this.serverIdList = [];

        this.serverUserCount = [];

    }
}

module.exports = LoginInfo;

File: Gun & Block/Assets/Script/Factory/LoginInfo.cs
using System.Collections.Generic;
using System.Collections;

class LoginInfo {

    // 0 正确 1密码错 2用户名错
    public int stateCode;
    public string msg;
    public int[] serverIdList;
    public int[] serverUserCountList;

    public LoginInfo(int stateCode, string msg) {

        this.stateCode = stateCode;

        this.msg = msg;

        // this.serverIdList = new int[100];

        // this.serverUserCount = new int[100];

    }
}